sdk/storage/Azure.Storage.Webjobs.Extensions.Samples/Filters/WorkItemValidatorAttribute.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+using SampleHost.Models;
+
+namespace SampleHost.Filters
+{
+    /// <summary>
+    /// Sample invocation filter that demonstrates how declarative validation logic
+    /// can be integrated into the execution pipeline.
+    /// </summary>
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
A bit strange that we have a sample that use obsolete members

Reply:
Logged work item to investigate this further. Good that our analyzers show that.